go.mod

```diff
@@ -0,0 +1,98 @@
+module github.com/pravega/pravega-operator
```

Comment:
Since this file is added, files  `Gopkg.lock` and `Gopkg.toml` can be removed.

Reply:
removed both of the files